File: Java/Disk.java
/* membuat class */ 
class Disk{

	/* deklarasi atribut private */
	private	String type;
	private int capacity;
	private int price;

	/* constructor */
	public Disk(){}

	public Disk(String type, int capacity, int price){
		this.type = type;
		this.capacity = capacity;
		this.price = price;
	}

	/* setter dan getter untuk atribut type */
	public void setType(String type){ 
		this.type = type;
	}

	public String getType(){
		return this.type;
	}

	/* setter dan getter untuk atribut capacity */
	public void setCapacity(int capacity){ 
		this.capacity = capacity;
	}

	public int getCapacity(){
		return this.capacity;
	}

	/* setter dan getter untuk atribut price */
	public void setPrice(int price){ 
		this.price = price;
	}

	public int getPrice(){
		return this.price;
	}

	/* method untuk mencetak output atribut dari class */ 
	public void printDisk(){
		System.out.println("Jenis Disk  : " + this.type + " " + this.capacity + " GB");
		System.out.println("Harga       : Rp" + this.price);
		System.out.println("---------------------------------------");
	}
}

File: Python/Disk.py
# membuat class
class Disk():

	# deklarasi atribut private
	__tipe = ""
	__capacity = 0
	__price = 0
	
	# constructor 
	def __init__(self):
		self.__tipe = ""
		self.__capacity = 0
		self.__price = 0

	# setter dan getter untuk atribut type
	def setType(self, tipe):
		self.__tipe = tipe

	def getType(self):
		return self.__tipe

	# setter dan getter untuk atribut capacity
	def setCapacity(self, capacity):
		self.__capacity = capacity

	def getCapacity(self):
		return self.__capacity

	# setter dan getter untuk atribut price
	def setPrice(self, price):
		self.__price = price

	def getPrice(self):
		return self.__price;
	
	# method untuk mencetak output atribut dari class 
	def printDisk(self):
		print("Jenis Disk  : {} {} GB".format(self.__tipe, self.__capacity))
		print("Harga       : Rp{}".format(self.__price))
		print("---------------------------------------")